src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
+    return element.src === img;
+  };
+
+  var tex = this.textures.find(f);
+  if (tex === undefined) {
+    tex = new p5.Texture(this, img);
+    this.textures.push(tex);
+  }
 
+  tex.set(img);
```

Comment:
So I'm just concerned that we're doing this repeatedly in a tight loop. Pretty sure we can limit what happens at draw time to just what's necessary.